horizons/engine/generate_atlases.py

```diff
@@ -404,7 +404,10 @@ def clear_everything(cls):
 			if not os.path.exists(path):
 				continue
 			cls.log.info('Deleting %s', path)
-			os.unlink(path)
+			try:
+				os.unlink(path)
+			except FileNotFoundError:
+				pass
```

Comment:
I've always learned that you should never have an empty catch statement.

I can see that it is reasonable here, but I think it would be better to at least log it

Reply:
The previous "continue" was not logging it either.

Reply:
Now that the other check is dropped, there might be more log entries from here.

Reply:
That's fair. Seeing this empty block feels wrong to me, but it's probably the best indeed.
Maybe add a comment that mentions what you just said: that this is the desirable state anyways